### PR TITLE
chore: Standardise pre-commits and workflows

### PR DESCRIPTION
**Pull request by baolsen**
Putting in our current standards for pre-commits, workflows, etc.

Excludes changes for tfsec and checkov, these have been disabled for now.

### CODE REVIEW

**Comment by baolsen** [comment]
Note, I had to remove AWS_ACCESS_KEY and AWS_SECRET_ACCESS_KEY secrets configuration from this repo for the checks to pass. It did not appear that these were being used anywhere, and the user they were associated with does not look correct.

**Comment by baolsen** [comment]
@phzietsman please take a look. Nothing too hectic here

File: .github/workflows/pr-title.yml [hunk 1]
```diff
@@ -49,4 +49,4 @@ jobs:
           # will suggest using that commit message instead of the PR title for the
           # merge commit, and it's easy to commit this by mistake. Enable this option
           # to also validate the commit message for one commit PRs.
-          validateSingleCommit: true
+          validateSingleCommit: false
```
Comment (phzietsman) [inline .github/workflows/pr-title.yml:52]: Should this not be true?

File: .tfdocs-config.yml [hunk 2]
```diff
@@ -15,29 +15,28 @@ sections:
...
   ----
-  {{ .Requirements }}   
+  {{ .Requirements }}
+
   ----
-  {{ .Resources }}  
+  {{ .Resources }}
+
   ----
-  ### Default excluded scoped actions
```
Comment (phzietsman) [inline .tfdocs-config.yml:37]: Revert these, need it on the README

**Review by phzietsman — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-title.yml (hunk 1 above)
Comment (baolsen) [reply .github/workflows/pr-title.yml:52]: Thanks, that's a bug copy-pasted from the template. I'll fix it there also. 

It's a [legacy configuration](https://github.com/amannn/action-semantic-pull-request#legacy-configuration) but I'll set it back to true, and add the other one that's mentioned.

File: .tfdocs-config.yml (hunk 2 above)
Comment (baolsen) [reply .tfdocs-config.yml:37]: Thanks, fixed up

**Review by phzietsman — APPROVED** [review]

**Pull request merged**